__tests__/jest.config.js

```diff
@@ -13,7 +13,7 @@ const config = {
       '<rootDir>/__tests__/transformer/imageTransformer.js'
   },
   transformIgnorePatterns: [
-    'node_modules/(?!((jest-)?react-native(-.*)?|@react-native(-community)?|p-timeout?|p-wait-for?|@notifee?)/)'
+    'node_modules/(?!((jest-)?react-native(-.*)?|@react-native(-community)?|p-timeout?|p-wait-for?|@notifee?)|@fengweichong/react-native-gzip?/)'
```

Comment:
Why is this edit needed? Seems like this is not related with the PR, is it?

Reply:
This is needed to run unit tests. Now that CozyWebView imports useLauncherContext, which ends up importing `src/libs/cozyAppBundle/cozyAppBundle.ts` which imports react-native-gzip.